documentation/shared/attributes.adoc

```diff
@@ -42,6 +42,7 @@
 //Monitoring links
 :GrafanaHome: link:https://grafana.com/[Grafana Labs^]
 :JMXExporter: link:https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter^]
+:StrimziMetricsReporter: link:https://github.com/strimzi/metrics-reporter[Strimzi Metrics Reporter^]
```

Comment:
do we want to link to this?  We only use the link in one place. Maybe we can live without it?

Reply:
I think there's another place that we should be linking to this, I've added a comment there, so maybe we should keep this?